i6_models/parts/conformer/convolution.py

```diff
@@ -28,20 +28,23 @@ class ConformerConvolutionV1(nn.Module):
     """
     Conformer convolution module.
     see also: https://github.com/espnet/espnet/blob/713e784c0815ebba2053131307db5f00af5159ea/espnet/nets/pytorch_backend/conformer/convolution.py#L13
+
+    Uses explicit padding for ONNX exportability, see:
+    https://github.com/pytorch/pytorch/issues/68880
     """
 
     def __init__(self, model_cfg: ConformerConvolutionV1Config):
         """
         :param model_cfg: model configuration for this module
         """
         super().__init__()
-
+        assert model_cfg.kernel_size % 2 == 1, "ConformerConvolutionV1 only supports odd kernel sizes"
```

Comment:
I thought the decision was that this check is in the cfg and not here? (Even though I disagreed with this decision.)

Reply:
Good point, why not something like this now?